src/click/formatting.py

```diff
@@ -129,13 +130,17 @@ def dedent(self):
         """Decreases the indentation."""
         self.current_indent -= self.indent_increment
 
-    def write_usage(self, prog, args="", prefix="Usage: "):
+    def write_usage(self, prog, args="", prefix=None):
         """Writes a usage line into the buffer.
 
         :param prog: the program name.
         :param args: whitespace separated list of arguments.
-        :param prefix: the prefix for the first line.
+        :param prefix: The prefix for the first line. Defaults to
+            ``"Usage: "``.
         """
+        if prefix is None:
+            prefix = _("Usage: ")
```

Comment:
trailing whitespace

Reply:
I couldn't tell if it was intentional that the prefix string contained the trailing space instead of doing `"{prefix}: "`. I don't mind making that change, but it might turn out weird if someone used a custom prefix like `"Usage -\t"`.

Reply:
prefix = _("Usage:") + " "

problem solved ;)

Reply:
So the colon should still be included?

Reply:
yeah I'd just avoid the whitespace